2015/proposals/agarwal-rajat.md

```diff
@@ -0,0 +1,92 @@
+# Title
```

Comment:
Replace with

~~~
# Add taxonomic name resolution to the EcoData Retriever to facilitate data-intensive ecology
~~~